tests/flows/test_vesting.py

```diff
@@ -345,13 +345,14 @@ def test_vesting_publisher_exit_scam(
     # other users have added liquidity, so the owner of the pool will have lower percentage
     assert pool_shares_before > bpool.balanceOf(publisher_addr) / bpool.total_supply()
 
+    # TODO: Commented because exit_pool contract method removed
```

Comment:
should this TODO remain? why comment out instead of delete?

Reply:
No this TODO should not remain.  I updated the test